docs/content/howtos/factory-reset.md

```diff
@@ -57,37 +57,40 @@ and configurations.
     ./InfectionMonkey-<VERSION>.AppImage
     ```
 
-1. Use your browser to navigate to `https://localhost:5000` (modify the host as
-   necessary) and register a new user.
+1. **Register a new user.**
+
+    Use your browser to navigate to `https://localhost:5000` (modify the host
+    as necessary) and register a new user.
 
 
 ### For Docker installations
 
-1. Stop the Monkey Island:
+1. **Stop the Monkey Island.**
     ```bash
     sudo docker kill monkey-island monkey-mongo
     ```
 
-1. Delete the MongoDB docker volume:
+1. **Delete the MongoDB docker volume.**
     ```bash
     sudo docker volume rm monkey-db
     ```
-1. Delete the Monkey Island container:
+1. **Delete the Monkey Island container.**
     ```bash
     sudo docker rm monkey-island
     ```
-1. Remove any other volumes that may be associated with Infection Monkey. You
-   can discover other volumes with the command:
+1. **Remove any other volumes that may be associated with Infection Monkey.**
+
+    You can discover other volumes with the command:
     ```bash
     sudo docker volume ls
     ```
 
-    You can remove the volumes with the command:
+    You can remove the volumes with the command.
```

Comment:
Typo? Seems to me a colon would be more appropriate here

Reply:
Yeah, that one shouldn't have been changed. I'll fix it.